lib/armbian-configng/config.ng.jobs.json

```diff
@@ -283,54 +285,225 @@
+		                    "doc_link": "",
+		                    "src_reference": "",
+		                    "author": "",
+		                    "condition": "! check_if_installed bluetooth && ! check_if_installed bluez && ! check_if_installed bluez-tools"
+                		},
+                        {
+                            "id": "SN01",
+                            "description": "Remove Bluetooth support",
+                            "command": [
+                                "see_current_apt ",
+                                "debconf-apt-progress -- apt-get -y remove bluetooth bluez bluez-tools",
+                                "check_if_installed xserver-xorg && debconf-apt-progress -- apt-get -y remove pulseaudio-module-bluetooth blueman",
+                                "debconf-apt-progress -- apt -y -qq autoremove"
+                            ],
+                            "status": "Pending Review",
+                            "doc_link": "",
+                            "src_reference": "",
+                            "author": "",
+                            "condition": "check_if_installed bluetooth || check_if_installed bluez || check_if_installed bluez-tools"
+                        },
+                        {
+                            "id": "SN02",
+                            "description": "Bluetooth Discover",
+                            "prompt": "Verify that your Bluetooth device is discoverable!",
+                            "command": [
+                                "connect_bt_interface"
+        
+                            ],
+                            "status": "Pending Review",
+                            "doc_link": "",
+                            "src_reference": "",
+                            "author": "",
+                            "condition": "check_if_installed bluetooth || check_if_installed bluez || check_if_installed bluez-tools"
+                        }
 		            ],
                     "status": "Active",
                     "doc_link": "",
                     "src_reference": "",
-                    "author": "",
-                    "condition": "check_if_installed bluetooth || check_if_installed bluez || check_if_installed bluez-tools"
+                    "author": ""
                 },
                 {
-                    "id": "N02",
-                    "description": "Bluetooth Discover",
-                    "command": [
-						"get_user_continue \"Verify that your Bluetooth device is discoverable!\" process_input ; connect_bt_interface"
-
+                    "id": "MN02",
+                    "description": "Configure network interfaces (NetworkManager)",
+                    "condition": "[[ -n $( grep -m1 'renderer: NetworkManager' /etc/netplan/*.yaml) ]]",
+                    "sub": [
+						{
+                            "id": "SN05",
+                            "description": "Select a Network Connection",
```

Comment:
After introducing of Netplan networking config by default, we can drop all direct use of Network Manager. IP, DNS, DHCP ... everything is defined via NetPlan, which passes it to either networkd or network manager. We need to detect it and pass a parameter `renderer` correctly. This is also backward compatible.

function:
etplan_wrapper "10-dhcp-all-interfaces" "ethernets" "**networkd|NetworkManager**" "${SELECTED_ADAPTER}" "$IP_ADDRESS""